pkgs/development/python-modules/behave/default.nix

```
@@ -16,22 +16,22 @@ buildPythonApplication rec {
     sha256 = "1ssgixmqlg8sxsyalr83a1970njc2wg3zl8idsmxnsljwacv7qwv";
   };
 
-  checkInputs = [ mock pathpy pyhamcrest pytest pytest-html ];
+  checkInputs = [ mock pathpy pyhamcrest pytest pytest-html pytestCheckHook ];
```

```suggestion
  checkInputs = [ pytestCheckHook mock pathpy pyhamcrest pytest-html ];
```
``pytest`` is redundant with ``pytestCheckHook``. Personal preference is to put ``pytestCheckHook`` first b/c it's the test runner.